lib/python/rose/task_env.py

```diff
@@ -81,16 +81,17 @@ def get_prepend_paths(event_handler=None, path_root=None, path_glob_args=[],
     if full_mode:
         for name, path_globs in PATH_GLOBS.items():
             path_globs_map[name] = path_globs
-    for path_glob_arg in path_glob_args:
-        if path_glob_arg is None:
-            continue
-        if "=" in path_glob_arg:
-            name, value = path_glob_arg.split("=", 1)
-        else:
-            name, value = "PATH", path_glob_arg
-        if name not in path_globs_map:
-            path_globs_map[name] = []
-        path_globs_map[name].append(value)
+    if path_glob_args:
```

Comment:
What is the function of this?

Reply:
The argument was set to the dangerous default `[]`. The default is now `None`, so need to use an `if` block to protect the loop.